### PR TITLE
John progress

### PR DESCRIPTION
**Pull request by jiunnyuan1991**

### CODE REVIEW

File: virtual-cluster/john.md [hunk 1]
```diff
@@ -152,15 +175,61 @@ You can download the iso file with:
...
+
+After installation, edit /etc/ssh/sshd_config and set PermitRootLogin and PasswordAuthentication as 'yes'.
+Then from the VM's node copy ssh key into the VM
+```
+ssh-copy-id 192.168.16.21
+```
+After copying over the ssh key, set PermitRootLogin to prohibit-password and PasswordAuthentication to 'no'.
+
+
+##Ansible Playbook
```
Comment (eugene-chow) [inline virtual-cluster/john.md]: The header is not rendered properly. Add a space after `##`. Do the same for the rest.

**Review by eugene-chow — CHANGES_REQUESTED** [review]
Instead of displaying the full URL, give it a meaningful title (or the title of the article) and embed the link in the title. For eg. [ Enabling Nested Virtualization for VM Instances](https://cloud.google.com/compute/docs/instances/enable-nested-virtualization-vm-instances). Most of the time, this is the best practice. But in instances such as `After setting the firewall rule, you can access the Proxmox VE user interface via https://YourPublicDomain.com:8006`, I would display the full URL. It's at your discretion to use the right approach.

@CloudCommandos/infra please take note

**Comment by jiunnyuan1991** [comment]
Dear Eugene, I have modified accordingly. Thanks!

**Review by eugene-chow — APPROVED** [review]
LGTM. Good work!

**Pull request merged**